hacs.json

```
@@ -0,0 +1,6 @@
+{
+    "name": "NOAA Tides and Currents",
+    "domains": ["sensor"],
+    "render_readme": true,
+    "filename": "sensor.py"
```

```suggestion
    "filename": "noaa_coops/sensor.py"
```

This would keep the path structure of the repo intact so it's the least likely to break anyone's setup.

If it isn't tenable for whatever reason, we can go with what you have here, and I'll just warn anyone who cloned into custom_components/ that they need to reclone into a subpath called noaa_coops

---

@jshufro No that does not work since there are only two options for HACS path structures:

1. custom_components/<integration_name>

2. all files in the root of the repository AND the 'content_in_root' set to True in hacs.json

There is no other path structures supported.

However, one option might be to use a symlink. This may not work on Windows based installations, but should on any Linux or Mac HA instances. Nope, that still did not pass the HACS structure check for integrations. 

HOWEVER, how about the reverse. Move noaa_tides into custom_components structure like most integrations that use HACS. Then symlink noaa_tides in root to the new path. This would allow people to install this manually still using whatever mechanism they previously did (e.g. clone, etc) *AND* also allow HACS to work.